### PR TITLE
Readability and usability style changes for installation guide.

### PR DESCRIPTION
**Pull request by benmcnelly**
Jumping around in the docs was hurting my eyes, made it closer to marketing page style, also optional graphic for page as well.

### CODE REVIEW

**Comment by astrelsky** [comment]
This looks really nice. The only potential problem I see is the following from https://github.com/NationalSecurityAgency/ghidra/pull/923#issuecomment-522976618

While not entirely related to the proposed changes it may be a good time to change "Platforms Supported" to "Supported Platforms" and remove lines 166 & 167 as that was likely an internal thing. (I just didn't see a reason to open an issue for the two minor things)

**Comment by benmcnelly** [comment]
@astrelsky sounds good, went ahead with those changes.

File: GhidraDocs/InstallationGuide.html [hunk 1]
```diff
@@ -1,581 +1,672 @@
...
-            </ol>
-          </li>
-          <li>
-            <p>Open Environment Variables window:</p>
+
+    <!--main role="main"-->
+    <main role="main">
+      <!-- Main jumbotron for a primary marketing message or call to action -->
+      <div class="jumbotron" style="text-align:center">
+	    <img style="width: 50%" src="images/GHIDRA_DOCS.png" alt="Ghidra" />
```
Comment (ryanmkurtz) [inline GhidraDocs/InstallationGuide.html]: Unfortunately like @astrelsky said we cannot accept the binary image.  So this line can come out too.

File: GhidraDocs/css/bootstrap.min.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*!
```
Comment (ryanmkurtz) [inline GhidraDocs/css/bootstrap.min.css]: We intentionally keep the installation guide as one file to make it easy to distribute outside of the Ghidra installation directory.  I wonder how invasive just pasting this entire file at the top of the installation guide would be...

File: GhidraDocs/InstallationGuide.html (hunk 1 above)
Comment (astrelsky) [reply GhidraDocs/InstallationGuide.html]: This image is one which aleady exists in the repo. [GhidraDocs/images/GHIDRA_1.png](https://github.com/NationalSecurityAgency/ghidra/blob/master/GhidraDocs/images/GHIDRA_1.png)

File: GhidraDocs/css/bootstrap.min.css (hunk 2 above)
Comment (benmcnelly) [reply GhidraDocs/css/bootstrap.min.css]: That would be not too bad.

**Pull request closed without merging**